main.go

```diff
@@ -46,7 +46,7 @@ type header struct {
 
 type Test struct {
 	Url     string       `json:"url,omitempty"`
-	Headers []header     `json:"headers,omitempty"`
+	Headers []header     `json:"header,omitempty"`
```

Comment:
I did not get why `headers` to `header`?

Reply:
As in test file, we pass header and not headers